pkg/neuvector-ui-ext/utils/workloads.ts

```diff
@@ -0,0 +1,29 @@
+import { getWorkloads } from '../plugins/workloads-class';
+import { Store } from 'vuex';
+import { cacheNvNamespace } from '../utils/common';
+
+export async function cacheWorkloadsInfo(store: Store<any>) {
+    await cacheNvNamespace(store);
+    let workloadRes = await getWorkloads();
+    store.dispatch('neuvector/updateWorkloads', preprocessNeuVectorWorkloadsData(workloadRes.data.workloads));
+}
+
+function preprocessNeuVectorWorkloadsData(workloads: any[]) {
+  let workloadKV = new Map();
```

Comment:
This seems unfinished? I presume we should use similar logic to `pkg/neuvector-ui-ext/utils/nodes.ts` to populate Map

Reply:
Yes, it is a pending function for getting full workload list from NeuVector API and preprocess it. But I need to confirm with Rancher team about performance concern. This is only a partial commit. This part is excluded.
After finalize the topic, we can determine if the feature is needed or not.

Reply:
@rushk014 Please let me know if there is any other concern.
If no, could you also review 2 commits after NVSHAS-9242? Then we can merge them and let QA continue testing  security event page. Thanks